src/community/advanced-data-card.tsx

```diff
@@ -0,0 +1,471 @@
+                        iconBackground = styles.cardAction,
+                        iconBackgroundInverse = styles.cardActionInverse,
+                    },
+                    index
+                ) =>
+                    Icon ? (
+                        <IconButton
+                            size={48}
+                            key={index}
+                            onPress={onPress}
+                            aria-label={label}
+                            className={styles.cardActionIconButton}
+                            style={{display: 'flex'}}
+                        >
+                            <div className={isInverse ? iconBackgroundInverse : iconBackground}>
+                                <Icon color={iconColor} size={iconSize} />
+                            </div>
+                        </IconButton>
+                    ) : (
+                        <div key={index} className={styles.cardActionIconButton} />
+                    )
+            )}
+        </Inline>
+    );
+};
+
+const useTopActions = (actions?: Array<CardAction>, onClose?: () => void) => {
+    const {texts} = useTheme();
+    const finalActions = actions ? [...actions] : [];
+
+    if (onClose) {
+        finalActions.push({
+            label: texts.closeButtonLabel,
+            onPress: onClose,
+            Icon: IconCloseRegular,
+        });
+    }
+
+    return finalActions;
+};
+
+type CardContentProps = {
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleAs?: string;
+    pretitleLinesMax?: number;
+    title?: string;
+    titleAs?: string;
+    titleLinesMax?: number;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+};
+
+const CardContent: React.FC<CardContentProps> = ({
+    headline,
+    pretitle,
+    pretitleAs = 'p',
+    pretitleLinesMax,
+    title,
+    titleAs = 'h3',
+    titleLinesMax,
+    subtitle,
+    subtitleLinesMax,
+    description,
+    descriptionLinesMax,
+}) => {
+    const {textPresets} = useTheme();
+    return (
+        <div>
+            <Stack space={4}>
+                {(headline || pretitle || title || subtitle) && (
+                    <header>
+                        <Stack space={4}>
+                            {headline}
+                            <Stack space={4}>
+                                {pretitle && (
+                                    <Text2
+                                        color={vars.colors.textPrimary}
+                                        truncate={pretitleLinesMax}
+                                        as={pretitleAs}
+                                        regular
+                                        hyphens="auto"
+                                    >
+                                        {pretitle}
+                                    </Text2>
+                                )}
+                                <Text
```

Comment:
please, use text4 for title as defines specs
<img width="951" alt="image" src="https://github.com/Telefonica/mistica-web/assets/6722153/469571f7-dade-4007-9560-a9fe4b13b3ad">